.azure-pipelines.yml

```diff
@@ -42,6 +42,7 @@ jobs:
 
   - script: |
       python -m pip install --upgrade pip
+      pip install -r .pip-2033.txt
```

Comment:
I would rather `pip install -e`, since that won't make a wheel.

However, why not use `flit` here?

Reply:
Though it would have to be `flit install -s` or `flit install --pth-file` for now